.github/workflows/python-linters.yml

```diff
@@ -1,4 +1,6 @@
-name: Code quality
+name: >-
+  🤖
+  Code quality
```

Comment:
nit: I'd prefer that we drop "Code Quality" from here. It'll make the description shorter in "Checks" shorter, leaving more room for information about the environment.

Reply:
Isn't it short enough? Also, it'd be weird on this page https://github.com/pypa/pip/actions. I just wanted to move emoji here from the job name so that job names would be shorter.